configs.py

```diff
@@ -17,40 +17,42 @@ def pip_install(module_name):
     return "python -m pip --default-timeout=1200 install --progress-bar off -U " + module_name
 
 
-configs = Variations([dict(name="Update Docker images", command=["make", "images"]),
-                      dict(name="Update Pylint", command=[python, "-m", "pip", "install",
-                                                          "-U", "--user", "--progress-bar", "off", "pylint"]),
-                      dict(name="Create virtual environment", command=[python, "-m", "venv", env_name]),
+configs = Configuration([dict(name="Update Docker images", command=["make", "images"]),
```

Comment:
Why not switch it to `Step` already?

Reply:
because I expected you to argue that switch dict->Step should be a separate PR from documentation change, since it impacts runtime